CSS/components/button.css

```diff
@@ -0,0 +1,57 @@
+.btn{
+    min-width: 5rem;
+    min-height: 1.5rem;
+    padding: 0.5rem;
+    background-color: var(--cta);
+    color: var(--button-text-color);
+    border-radius: 0.2rem;
+}
+.btn i{
+    font-size: 1.5rem;
+}
+.btn-secondary{
+    border: 3px solid var(--cta);
```

Comment:
Dont define color name as CTA. Use words such as primary to define

Reply:
changed the color name